src/main/java/com/parrit/entities/PairingBoard.java

```diff
@@ -30,13 +30,18 @@
     @JoinColumn(name = "pairing_board_id", nullable = true, foreignKey = @ForeignKey(name = "pairing_board_id_fk"))
     private List<Person> people;
 
+    @OneToMany(cascade = CascadeType.ALL)
```

Comment:
When a role gets deleted, it currently is still present in the database, with a null `pairing_board_id` column.  I don't think we currently need to keep a row for these deleted roles, so please add this to the `@OneToMany` annotation so that hibernate actually deletes the row when a role is deleted: `orphanRemoval = true`